portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,124 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Fiana Vicker's Professional Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body = style = "border-top: lightslategray;
+                   background-color: grey; background-image: url('images/wooddark.jpeg');
+                   
+                   " >
+
+    <div id="banner">
+        <h2 style="top: 0px; background-color: palevioletred; color: lightgray;">
+                Personal Projects        |        Gallery       |          Professional Projects      
+                   |         Resume        |  Instagram  | 
+                   
+                   <a href="https://github.com/fianaVickers">GitHub </a> |  
+                   <a href="https://www.linkedin.com/in/fiana-vickers-ba51a3170/">LinkedIn  </a> </h2>
+    </div>
+
+
+    <div id="content">
+        
+      <h1 = style="margin-top: 0.25in; font-size: 65pt; color: lightgrey; font-style: italic;"> Fiana Vickers </h1>
+      <p style="font-style: italic; font-size: 20pt; color: lightgray;"> An aspring Software Engineer </p>
+
+    </div>               
+
+    <div id="rounded-borders">
+      <img src="images/flowers.jpg" >
```

Comment:
I noticed this file was a few MB. I'd consider scaling it down in the future. No matter how you format it, the browser has to download the whole file so it'll speed up your load times and use less network data if it's a little bit smaller.

Reply:
To clarify, I'm talking about the image file flowers.jpg